### ISSUE TITLE
Flag duplicate properties

### ISSUE THREAD

**Issue by gajus** [gajus/eslint-plugin-flowtype#259]
```js
type StatePropsType = {|
  +events: $PropertyType<MovieEventsViewStateType, 'events'>,
  +events: $PropertyType<MovieEventsViewStateType, 'filteredEvents'>,
|};

```

This produces no warning at the moment.

**Comment by gajus** [comment]
Closing as stale.
